test/verify/check-packagekit

```diff
@@ -271,6 +271,22 @@ echo -e "Loaded patch modules:\nkpatch_3_10_0_1062_1_1 [enabled]\n\nInstalled pa
         b = self.browser
         m = self.machine
 
+        def check_status(b, m):
+            if m.image in OSesWithoutTracer:
+                b.wait_in_text("#status p", "System is up to date")
+                # PK starts from a blank state, thus should force refresh and set the "time since" to 0
+                b.wait_in_text("#last-checked", "Last checked: less than a minute ago")
+            else:
+                code = m.execute("tracer > /dev/null 2>&1; echo $?").rstrip()
+                if code == "104":
+                    b.wait_in_text("#status p", "a system reboot")
+                elif code == "102" or code == "101":
+                    b.wait_in_text("#status p", "to be restarted")
+                elif code == "0":
+                    b.wait_in_text("#status p", "System is up to date")
+                    # PK starts from a blank state, thus should force refresh and set the "time since" to 0
+                    b.wait_in_text("#last-checked", "Last checked: less than a minute ago")
```

Comment:
Should this be checked only when tracer returns 0? I guess we want to check it every time, so it can go outside of all the conditions.

Reply:
Outside of all conditions? That would cause that in some cases, like when code 104 is returned, the test would check for `wait_in_text("#status p", "a system reboot")` and `wait_in_text("#status p", "System is up to date")`
So I think it should be in `else` condition

Reply:
I think @marusak just refered to teh last line the "last checked". That should be common for all scenarios.

Reply:
Indeed, I just meant the `Last checked: less than a minute ago` one.

Reply:
But the "last checked" line is not present for other scenarios (e.g. system needs reboot, some services need restart). It's only present for "System is up to date" and "# updates available" states.
And that's correct. Because if you have the latter two states, you may want to rerun RefreshCache to check if there are any new updates to the system.
But if the state is tracer telling you that your system needs reboot, then rerunning RefreshCache won't change a thing, and "last checked" is not really needed to be shown.

Reply:
I see, thanks for the explanation!